puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -9,6 +9,8 @@
   $backend_glusterfs_name = 'glusterfs',
   $backend_iscsi          = false,
   $backend_iscsi_name     = 'iscsi',
+  $backend_netapp         = false,
```

Comment:
On 20/01/15 22:47 -0800, Ryan Hefner wrote:
>   [2]@jguiditta thanks for taking a look through this. I'm not entirely
>   familiar with astapor, so help me understand what your comment above
>   means. I have added parameters above to
>   puppet/modules/quickstack/manifests/cinder_volume.pp, which seems to be
>   the necessary changes to quickstack::cinder. Are you suggesting that I
>   remove these parameters or add them to a different file? Thanks!
>
So, the changes you made to cinder_volume.pp look fine.  However, that
is called by quickstack::cinder, which in turn is configured by
pacemaker::cinder.  The relevant param changes were not present in
those two files, so this will not work as is.  Staypuft (or ofi core
hostgroups) use only pacemaker::cinder directly for juno, so that is
the entrypoint that needs to be covered here.  Please let me know if
that does not clarify what I was referring to in the previous comment.

-j